ui/src/app/views/app/component.html

```diff
@@ -7,6 +7,7 @@
 <nav *ngIf="(user | async).isSignedIn" style="margin-top: 1rem;">
   <a routerLink="/login" routerLinkActive="active">Switch User</a>
   <a routerLink="/repository" routerLinkActive="active">Select CDR</a>
+  <a routerLink="/workspace" routerLinkActive="active">My Workspace</a>
```

Comment:
Let's update this url to include a namespace and id, like `:ns/workspaces/:id` so there's a clear correspondence with our API and FireCloud?